test/test-cases/functional/saic/sai-api/test_sai_api_vnet_eni.py

```diff
@@ -0,0 +1,184 @@
+                    "SAI_ENI_ATTR_OUTBOUND_V4_STAGE3_DASH_ACL_GROUP_ID",
+                    "0",
+                    "SAI_ENI_ATTR_OUTBOUND_V4_STAGE4_DASH_ACL_GROUP_ID",
+                    "0",
+                    "SAI_ENI_ATTR_OUTBOUND_V4_STAGE5_DASH_ACL_GROUP_ID",
+                    "0",
+                    "SAI_ENI_ATTR_OUTBOUND_V6_STAGE1_DASH_ACL_GROUP_ID",
+                    "0",
+                    "SAI_ENI_ATTR_OUTBOUND_V6_STAGE2_DASH_ACL_GROUP_ID",
+                    "0",
+                    "SAI_ENI_ATTR_OUTBOUND_V6_STAGE3_DASH_ACL_GROUP_ID",
+                    "0",
+                    "SAI_ENI_ATTR_OUTBOUND_V6_STAGE4_DASH_ACL_GROUP_ID",
+                    "0",
+                    "SAI_ENI_ATTR_OUTBOUND_V6_STAGE5_DASH_ACL_GROUP_ID",
+                    "0"
+                ]
+            },
+        ]
+        results = [*dpu.process_commands(commands)]
+        print("\n======= SAI commands RETURN values create =======")
+        pprint(results)
+
+        assert all(results), "SAI_OBJECT_TYPE_ENI Create error"
+
+    @pytest.mark.skip(reason="get and set not implemented, yet")
+    def test_vnet_eni_get1(self, dpu):
+
+        commands = [
+            {
+                "name": "eni_id",
+                "op": "get",
+                "type": "SAI_OBJECT_TYPE_ENI",
+                "atrribute": "SAI_ENI_ATTR_VM_UNDERLAY_DIP"
+            }
+        ]
+        results = [*dpu.process_commands(commands)]
+        print("\n======= SAI commands RETURN values get =======")
+        pprint(results)
+
+        assert all( [result == 0 for result in results]), "10.10.2.10"
+
+    @pytest.mark.skip(reason="get and set not implemented, yet")
+    def test_vnet_eni_set(self, dpu):
+
+        commands = [
+            {
+                "name": "eni_id",
+                "op": "set",
+                "type": "SAI_OBJECT_TYPE_ENI",
+                "attribute": [
+                    "SAI_ENI_ATTR_VM_UNDERLAY_DIP",
+                    "20.10.2.10",
+
+                ]
+            },
+        ]
+        results = [*dpu.process_commands(commands)]
+        print("\n======= SAI commands RETURN values set =======")
+        pprint(results)
+
+        assert all( [result == 0 for result in results]), "SAI_OBJECT_TYPE_ENI Set error"
+
+    @pytest.mark.skip(reason="get and set not implemented, yet")
+    def test_vnet_eni_get2(self, dpu):
+
+        commands = [
+            {
+                "name": "eni_id",
+                "op": "get",
+                "type": "SAI_OBJECT_TYPE_ENI",
+                "atrribute": "SAI_ENI_ATTR_VM_UNDERLAY_DIP"
+
+            }
+        ]
+        results = [*dpu.process_commands(commands)]
+        print("\n======= SAI commands RETURN values get =======")
+        pprint(results)
+
+        assert all( [result == 0 for result in results]), "20.10.2.10"
+
+    #@pytest.mark.dependency(depends=['test_sai_api_vnet_007_in_route.py::test_vnet_inbound_routing_entry_create'], scope='session')
```

Comment:
IMO this is a good idea to use `@pytest.mark.dependency()` here. Otherwise on create failure we will have delete failure as well. Good unless a create happens but RC check fails ;)

Reply:
fixed

Reply:
This will be added as a new PR where dependencies will be addressed